test/headless/src/extension/xapi.ts

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios'
+
+const {XAPI_URL = '', XAPI_PASS = '', XAPI_USER = ''} = process.env
```

Comment:
Put these in the new `test/config` module?

Reply:
Done